aten/src/ATen/native/quantized/cpu/qadd.cpp

```diff
@@ -48,35 +48,37 @@ Tensor _add_out(Tensor& out, const Tensor& self, const Tensor& other) {
 
   AT_DISPATCH_QINT_TYPES(out.scalar_type(), "qadd", [&]() {
     using Vec = Vec256<scalar_t>;
```

Comment:
Do these files have any changes apart from clang format? I think we shouldn't include them in this PR as they don't use qnnpack.

Reply:
Thanks, done.